ultra_rest_client/ultra_rest_client.py

```diff
@@ -179,6 +179,31 @@ def get_zones(self, q=None, **kwargs):
         params = build_params(q, kwargs)
         return self.rest_api_connection.get(uri, params)
 
+    # list zones for all user accounts using v3 url
+    def get_zones_v3(self, q=None, **kwargs):
+        """Returns a list of zones across all of the user's accounts.
+
+        Keyword Arguments:
+        q -- The search parameters, in a dict.  Valid keys are:
+             name - substring match of the zone name
+             zone_type - one of:
+                PRIMARY
+                SECONDARY
+                ALIAS
+        sort -- The sort column used to order the list. Valid values for the sort field are:
+                NAME
+                ACCOUNT_NAME
+                RECORD_COUNT
+                ZONE_TYPE
+        reverse -- Whether the list is ascending(False) or descending(True)
+        offset -- The position in the list for the first returned element(0 based)
```

Comment:
Would `offset` parameter still be in `v3`, since we switched to cursor?

Reply:
Yeah, its no more required. Removed the same.